scripts/us_census/pep/annual_population/preprocess_test.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Script to automate the testing for USA Population preprocess script.
+"""
+
+import os
+import unittest
+from os import path
+from preprocess import USCensusPEPAnnualPopulation
+# module_dir_ is the path to where this test is running from.
+module_dir_ = os.path.dirname(__file__)
+test_data_folder = os.path.join(module_dir_, "test_data")
+op_data_folder = os.path.join(module_dir_, "test_output_data")
+
+
+class TestPreprocess(unittest.TestCase):
+    """
+    TestPreprocess is inherting unittest class
+    properties which further requried for unit testing
+    """
+
+    cleaned_csv_file_path = os.path.join(op_data_folder, "test_output_data.csv")
```

Comment:
Pls add tests for each type of input format: txt for different year ranges, csv

Reply:
Please add a test for the .txt input file format? I don't see any .txt in test_data/